wikitools/category.py

```diff
@@ -106,10 +106,11 @@ def __getMembersInternal(self, namespaces=False):
 			params['cmnamespace'] = '|'.join([str(ns) for ns in namespaces])
 		while True:
 			req = api.APIRequest(self.site, params)
-			data = req.query(False)
+			#data = req.query(True)
```

Comment:
Please don't leave commented out code in the file. We have Git history to know how things were previously. :-)